scaleway/provider.go

```diff
@@ -78,8 +99,20 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		}
 	}
 
+	organization := d.Get("organization").(string)
+
+	if apiKey == "" {
+		if path, err := homedir.Expand(d.Get(".scwrc").(string)); err == nil {
+			scwAPIKey, scwOrganization, err := readScalewayConfig(path)
+			if err == nil {
```

Comment:
should we invert this and raise this error?
```suggestion
			if err != nil {
			  return fmt.Errorf("Error loading credentials from SCW: %s", err)
			}
```

Reply:
that's a miss, yes it should be inverted